clap_complete/src/dynamic/shells/mod.rs

```diff
@@ -27,6 +107,70 @@ pub enum CompleteCommand {
 }
 
 /// Generally used via [`CompleteCommand`]
+///
+/// This is the arguments for the `complete` subcommand.
```

Comment:
Someone using `CompleteCommand` will likely not look at `CompleteArgs` to see documentation on how to use this